articles/defender-for-cloud/devops-faq.md

```diff
@@ -164,6 +165,14 @@ We don’t support automatic migration for the Defender for DevOps connectors fr
 
 If you want to move a connector’s location, for example a GitHub or Azure DevOps connector, to be stored in a different region than the original one where the connector was created, the recommendation is to delete the existing connector and then to create another connector in the new region.
 
+### How can I see the MSDO scan results in GitHub
+
+Follow the guidance to set up the GitHUb Action [here](https://learn.microsoft.com/en-us/azure/defender-for-cloud/github-action).
+
+Ensure that you have Read and Write permissions set on the workflow permissions in the GitHub repository. See more [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#configuring-the-default-github_token-permissions).
```

Comment:
Can we add a note to use with cautious for public repos and it give read and write permission on the repo